plugins/org.yakindu.sct.ui/src/org/yakindu/sct/ui/wizards/ModelCreationWizardPage.java

```diff
@@ -70,6 +70,11 @@ protected boolean validatePage() {
 			setErrorMessage(NLS.bind("file extension is not valid! The valid extension is: " + "." + fileExtension, extension));
 			return false;
 		}
+		String name = getFileName();
+		if (name != null && !name.matches("^[-a-zA-Z0-9._]+")) {
+			setErrorMessage("file name is not valid!");
```

Comment:
You can reuse the error msg from line 70

Reply:
But the error that is catched in these lines has nothing to do with the extension, which could lead to users being confused, because they have the "`.sct`"-extension;
for example the name "`test file.sct`" which gets translated to "`test%20file.sct`" which is invalid not because of its extension but because it contains a space(or just an invalid name in generel)

Reply:
I agree with @Mazesch